pkg/dnshelper/dnshelper.go

```diff
@@ -0,0 +1,46 @@
+package dnshelper
+
+import (
+	"fmt"
+	"html/template"
+
+	"github.com/massalabs/station/pkg/convert"
+	"github.com/massalabs/station/pkg/node"
+)
+
+const (
+	metaKey = "META"
+
+	// Indexes of data in website name key.
+	indexOfWebsiteAddress     = 0 // Index of website Address in the dnsValue array
+	indexOfWebsiteDescription = 2 // Index of website Description in the dnsValue array
+)
+
+// Helper function to fetch website values from DNS entry.
```

Comment:
As you function signature is ambiguous, you should describe the returned values explaining that the first one is an address and the second on is the description content.

Furthermore, you should avoid using word such as get or fecth we you don't grab the infomration from a remote service. here you are simply converting it.

Something as simple as the following should be sufficent:

```suggestion
// AddressAndDescription fetch the website address and it's description from the DNS entry.
```